backend/dal/Configuration/AccessRequestAgencyConfiguration.cs

```diff
@@ -1,28 +1,45 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
 using Pims.Dal.Entities;
+using Pims.Dal.Extensions;
 
 namespace Pims.Dal.Configuration
 {
     /// <summary>
     /// AccessRequestAgencyConfiguration class, provides a way to configure AccessRequest Agencies in the database.
     /// </summary>
-    public class AccessRequestAgencyConfiguration : BaseEntityConfiguration<AccessRequestAgency>
+    public class AccessRequestAgencyConfiguration : BaseAppEntityConfiguration<AccessRequestAgency>
     {
         #region Methods
         public override void Configure(EntityTypeBuilder<AccessRequestAgency> builder)
         {
-            builder.ToTable("AccessRequestAgencies");
+            builder.ToMotiTable().HasAnnotation("ProductVersion", "2.0.0");
```

Comment:
so this annotation is the version that this database entity was added in (and therefor shouldn't change)?

Reply:
This is one of the annoying things I need to remove.  EF Core 5 doesn't correctly apply all annotations, which means this does nothing.  However the purpose was to identify when the table was added.